app/js/controllers/admin/why-nijel.js

```diff
@@ -0,0 +1,89 @@
+const AdminWhyNijelCtrl = function ($scope, $state, $mdDialog, $mdToast, AdminDataService, ClientDataService, $sce) {
+
+    $scope.sections = getSections();
+    $scope.selected = [];
```

Comment:
Same again, here $scope.selected isn't being used

Reply:
Done. Made the change to remove the unused variable